### PR TITLE
Camera iris transition animation (without hollow state)

### PR DESCRIPTION
**Pull request by tbaranes**
#126  WIP

I missed it yesterday, here it is: system camera iris transition

### CODE REVIEW

File: IBAnimatableApp/TransitionTableViewController.swift [hunk 1]
```diff
@@ -50,7 +50,7 @@ private extension TransitionTableViewController {
     transitionAnimationsHeaders.append("Suck Effect")
     transitionAnimations.append(["SystemSuckEffect"])
     transitionAnimationsHeaders.append("Camera Iris")
-    transitionAnimations.append(["SystemCameraIris(Open)", "SystemCameraIris(Close)"])
+    transitionAnimations.append(["SystemCameraIris", "SystemCameraIris(HollowOpen)", "SystemCameraIris(HollowClose)"])
```
Comment (JakeLin) [inline IBAnimatableApp/TransitionTableViewController.swift:53]: do we need to add `SystemCameraIris` to the end like `transitionAnimations.append(["SystemCameraIris", "SystemCameraIris(HollowOpen)", "SystemCameraIris(HollowClose)", "SystemCameraIris"])`

**Comment by JakeLin** [comment]
:+1:  I didn't know the `.None` option either.

File: IBAnimatableApp/TransitionTableViewController.swift (hunk 1 above)
Comment (tbaranes) [inline IBAnimatableApp/TransitionTableViewController.swift:53]: Why should we?

Comment (JakeLin) [inline IBAnimatableApp/TransitionTableViewController.swift:53]: @tbaranes sorry, `SystemCameraIris` is the first one, we don't extra one.

Comment (JakeLin) [inline IBAnimatableApp/TransitionTableViewController.swift:53]: @tbaranes sorry, `SystemCameraIris` is the first one, we don't need the extra one.

**Pull request merged**